addon/templates/components/frost-fixed-table.hbs

```diff
@@ -31,8 +31,10 @@
       hook=(concat hookPrefix '-header-left')
       hookQualifiers=hookQualifiers
       tagName='div'
+      isSelectable=_isSelectable
```

Comment:
This was done in frost-table, for sorting was my guess as well but will look a little deeper to confirm